assets/js/phoenix_live_view.js

```diff
@@ -1443,6 +1449,8 @@ export class View {
   }
 
   submitForm(form, targetCtx, phxEvent){
+    // touch all text areas to fix isEqualNode failing to use text area values
+    DOM.all(form, "textarea", el => el.setAttribute(PHX_TOUCHED, true))
```

Comment:
@snewcomer I added back the `isEqualNode` optimization, but I had to include this workaround here because it appears `textarea1.isEqualNode(textarea2)` does not take into account the textarea value. They both report their `innertHTML` to be empty values, so perhaps this is related? I'm not sure if it's a browser limitation or a morphdom bug we need to address, but would love your thoughts.